src/functions-and-arrays.js

```diff
@@ -26,13 +67,40 @@ function sum() {}
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function averageNumbers(numbersAvg) {
+
+  let average = 0;
+
+  if(numbersAvg==""){
+    return null;
+  }
+
+  else{
+    for (i=0; i<numbersAvg.length; i++){
+      average += numbersAvg[i];
```

Comment:
Here you are calculating the sum, not the average, so you could use a different variable with a more explicit name, like `total`.